### PR TITLE
[py] Make it possibly to specify known peers upon instantiation

### PR DESCRIPTION
**Pull request by abhinav**
@blampe @breerly @junchaowu 

This will make the autobahn stuff simpler

### CODE REVIEW

File: python/tchannel/tornado/peer.py [hunk 1]
```diff
@@ -26,7 +26,6 @@
 from random import random
 
 from tornado import gen
-
```
Comment (blampe) [inline python/tchannel/tornado/peer.py]: wut

Comment (abhinav) [inline python/tchannel/tornado/peer.py]: PyCharm auto format. Fixing.

File: python/tchannel/tornado/tchannel.py [hunk 2]
```diff
@@ -86,6 +87,10 @@ def __init__(self, hostport=None, process_name=None):
         self.event_emitter = EventEmitter()
         self.hooks = EventRegistrar(self.event_emitter)
 
+        if known_peers:
```
Comment (blampe) [inline python/tchannel/tornado/tchannel.py]: If I have an existing `TChannel` do I need to this myself?

Comment (abhinav) [inline python/tchannel/tornado/tchannel.py]: Yes

Comment (abhinav) [inline python/tchannel/tornado/tchannel.py]: .peers is part of the TChannel interface. Accessing it and adding/getting peers is completely okay.

**Comment by abhinav** [comment]
Added a quick test

**Comment by blampe** [comment]
lgtm

**Pull request merged**